Assets/HoloToolkit/Input/Scripts/Utilities/MotionControllerVisualizer.cs

```diff
@@ -262,18 +261,32 @@ private IEnumerator LoadControllerModel(InteractionSource source)
 
                     reader.ReadBytes(fileBytes);
                 }
+#else
+                IntPtr controllerModel = new IntPtr();
+                ulong outputSize = 0;
```

Comment:
ulong would match up to LONGLONG in C++, but your plugin is UINT32.  Potential overflow

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/types

Reply:
Oof, looks like I made some changes while testing that didn't get reverted. Thanks for catching!